packages/web/package.json

```diff
@@ -52,6 +57,7 @@
     "mobx-react-lite": "^3.2.2",
     "mobx-utils": "^6.0.4",
     "next": "12.0.10",
+    "next-transpile-modules": "9",
```

Comment:
What is this used for?

Reply:
ah it was used to make visx work with next.js - no longer needed as i've pinned visx version to one that's supported.